tests/install/openqa_webui.pm

```diff
@@ -74,8 +74,9 @@ sub install_containers {
 }
 
 sub run {
-    send_key "ctrl-alt-f3";
-    assert_screen "inst-console";
+    #send_key "ctrl-alt-f3";
+    #assert_screen "inst-console";
+    send_key_until_needlematch("inst-console", "ctrl-alt-f3", 5, 1);
```

Comment:
I would expect [send_key_until_needlematch](http://open.qa/api/testapi/#_send_key_until_needlematch) to hit they key up to 5 times with a 1 second delay.
Didn't the failure show that the console got opened? So we might just need a >30s timeout for `assert_screen`?

Reply:
Yes, but only if send_key works perfectly. Anyway, I change that assuming that the send_key is not the problem

Reply:
I changed that and I created this test https://openqa.opensuse.org/tests/1868785

Reply:
You changed it to `timeout => 30` - that is the default, so it would be exactly the same :-D